Don't look for and use programs we haven't declared dependencies on.
--- Makefile.orig	2024-01-16 22:03:20.000000000 -0600
+++ Makefile	2024-02-14 01:35:28.000000000 -0600
@@ -205,30 +205,10 @@
 
 clrnamdb: clrnamdb.scm
 clrnamdb.scm: mkclrnam.scm color.scm resenecolours.txt saturate.txt nbs-iscc.txt
-	if type scm; then scm -e"$(MKNMDB)";\
-	elif type guile; then guile -l guile.init -c\
-	 "(use-modules (ice-9 slib)) $(MKNMDB)";\
-	elif type slib48; then echo -e "$(MKNMDB)\n,exit" | slib48 -h 3000000;\
-	elif type umb-scheme; \
-	then SCHEME_INIT=`pwd`/umbscheme.init;export SCHEME_INIT;\
-	 echo "$(MKNMDB)" | umb-scheme;\
-	elif type mzscheme; \
-	then SCHEME_LIBRARY_PATH=`pwd`/;export SCHEME_LIBRARY_PATH;\
-	 echo "$(MKNMDB)" | mzscheme -f mzscheme.init;\
-	fi
+	-# no
 
 catalogs:
-	-if type scm; then scm -c "(require 'new-catalog)"; fi
-	-if type guile; then guile -l guile.init -c\
-	  "(use-modules (ice-9 slib)) (require 'new-catalog)"; fi
-	-if type umb-scheme; then\
-	  SCHEME_INIT=umbscheme.init;export SCHEME_INIT;\
-	  echo "(require 'new-catalog)" | umb-scheme; fi
-	-if type mzscheme; then\
-	  SCHEME_LIBRARY_PATH=`pwd`/;export SCHEME_LIBRARY_PATH;\
-	  cp mkpltcat.scm `mzscheme -mf mzscheme.init -e '(begin(display(implementation-vicinity))(exit))'`mkimpcat.scm;\
-	  mzscheme -g -f mzscheme.init -e "(require 'new-catalog)"</dev/null; fi
-	-if type scheme48; then $(MAKE) install48; fi
+	-# no
 
 $(DESTDIR)$(S48SLIB)strport.scm: $(DESTDIR)$(S48SLIB)
 	echo ";;; strport.scm  -*- scheme -*-">$(DESTDIR)$(S48SLIB)strport.scm
